code/game/mecha/mecha.dm

```diff
@@ -664,6 +664,10 @@
 		user << "<span class='warning'>Access denied.</span>"
 		log_append_to_last("Permission denied.")
 		return
+	if(user)(mob_buckled)

```

Comment:
what you've written already isn't valid syntax (and byond actually complained in the Travis report)